modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultCodegen.java

```diff
@@ -1452,6 +1452,10 @@ private static String getPrimitiveType(Schema schema) {
         } else if (ModelUtils.isUUIDSchema(schema)) {
             return "UUID";
         } else if (ModelUtils.isStringSchema(schema)) {
+            if(typeMapping.containsKey(schema.getFormat()))
+            {
```

Comment:
Can this open curly bracket go to line 1455 instead to conform to Java code style?

Reply:
Fixed and updated